src/GovUk.Education.ExploreEducationStatistics.Notifier/PublicationNotifier.cs

```diff
@@ -48,32 +52,36 @@ public class PublicationNotifier
         [FunctionName("PublicationNotifier")]
         // ReSharper disable once UnusedMember.Global
         public void PublicationNotifierFunc(
-            [QueueTrigger(PublicationQueue)] PublicationNotificationMessage publicationNotification,
+            [QueueTrigger(PublicationQueue)] PublicationNotificationMessage notificationMessage,
```

Comment:
Prior to this change the code only sent emails at a publication level regardless of whether  multiple releases were published within the same publication on the same evening.

The email contained the link to the latest release of the Publication.

I see that this PR is an overall change to now send emails for each Release version being published  so I'm not sure about the whole naming of this function, queue and message type any longer.

It was a originally a `PublicationQueue` which received a `PublicationNotificationMessage ` when a Publication had any new releases but that's no longer the case, so the naming may no longer be appropriate.

What happens now if there are two original releases of a publication being published on the same evening?

As th

Reply:
Discussed this with Ben on Slack. We agreed to reluctantly leave the name as it is, as despite the loss of clarity, there are issues with having to manually delete resources across environments as this is deployed, and having to ask for those resources to be deleted on Prod. Also, it is potentially easy to miss a reference to the resources being renamed that could introduce a bug.

Not sure what others think?

Reply:
As mentioned on the call today, having thought about it, there shouldn't be ARM template config to clean up or resources to delete in Azure if renaming the `PublicationNotifier` function because individual functions inside the function project have no specific config as far as I know. 

The new queue should create itself when first accessed. We can remove the unused queue once the change is deployed,(checking its empty first).

Reply:
> Seems like the more appropriate name would be release-notifications (or similar - also note that I've not added queue into the name as that's just redundant).

I've renamed the queue as you suggest, but I've named the variable in `NotifierQueues` to `ReleaseNotificationQueue`. With people normally importing classes like `NotifierQueues` statically, having suffixing `Queue` seem worth it in this instance.

Reply:
Done